### DISCUSSION TITLE
将连接到网络的名称为默认的iPhone识别为？的问题

### DISCUSSION THREAD

**Discussion by najloa** [immortalwrt/immortalwrt#1020] [Q&A]
不是什么特别影响使用的问题，但老的网件都能识别出是什么设备来
![屏幕截图](https://github.com/immortalwrt/immortalwrt/assets/15027661/3563f309-8005-4266-a892-d3ce1cd80834)
![捕获](https://github.com/immortalwrt/immortalwrt/assets/15027661/0aab0761-b8ee-4b4f-80c5-df1b4e6dd39d)

**Comment by 1715173329** [comment]
把随机 mac 地址关掉即可显示。
或者你的设备本身没有发送主机名。

**Reply by dux2468** [reply]
![image](https://github.com/immortalwrt/immortalwrt/assets/61663064/fd1a0c2b-c09a-45aa-a931-51dd5b08bf9c)
把私有地址关了就行了
